plugins/storage/volume/storpool/src/main/java/com/cloud/hypervisor/kvm/storage/StorPoolStoragePool.java

```diff
@@ -16,6 +16,7 @@
 // under the License.
 package com.cloud.hypervisor.kvm.storage;
 
+import com.cloud.hypervisor.kvm.resource.LibvirtVMDef;
```

Comment:
Nitpicking, but could you move this import with the other `com.` imports?

Reply:
Thanks, @BryanMLima, for the review! I made the change